### DISCUSSION TITLE
Is there a lower bound for sphere size?

### DISCUSSION THREAD

**Discussion by eferreirafilho** [NVlabs/curobo#149] [Software Q&A] [answered]
First, thanks for the great work you are doing! cuRobo is impressively fast!

I have a custom end effector that has a very thin part, which is crucial for my task. I have used the lula description tool to create the surrounding spheres, but they were too big. I manually edited them to be smaller, but the end effector has always a big gap to the obstacles which makes it useless to my task. I'm using spheres with radius between 0.0002 and 0.001, and I have set all the buffers to zero. Is this allowed in your approach?

Alternatively, is it possible to use other shapes as collision masks instead of spheres?

Thanks in advance!

**Comment by balakumar-s** [comment, marked as answer]
There is no lower bound. We use float32 for sphere radius, so that would be the limitation. We have tried up to 0.0001 and found no issues.

Currently, we do not support other collision representations for the robot.
